code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -152,6 +156,11 @@
 		use_power = 0
 	else if(!stat && construction_state <= 3)
 		use_power = 1
+	src.update_icon()

```

Comment:
I'm just going to go ahead and remove 'src.' everywhere it's implied in the 3 files this PR touches then, may as well